core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.spec.ts

```diff
@@ -577,4 +595,12 @@ describe('TopNavigationBarComponent', () => {
       component.isHackyTopicTitleTranslationDisplayed(0);
     expect(hackyStoryTitleTranslationIsDisplayed).toBe(true);
   });
+
+  it('should show android button if the feature is enabled', () => {
+    mockPlatformFeatureService.status.AndroidBetaLandingPage.isEnabled = true;
+
+    const component = TestBed.createComponent(TopNavigationBarComponent);
+
+    expect(component.componentInstance.androidPageIsEnabled).toBeTrue();
```

Comment:
Can we have a pre-check for this? If not can you add a comment explaining why it;s not possible

Reply:
Please make a similar change in all other places too

Reply:
The value for androidPageIsEnabled actually gets set on construction and is not modified after that, so I went ahead and added a comment explaining this for now.